readme.md

```diff
@@ -120,9 +128,9 @@ The `child_process` instance is enhanced to also be promise for a result object
 
 ### execa.sync(file, [arguments], [options])
 
-Execute a file synchronously.
+Execute a file synchronously. 
 
-Returns the same result object as [`child_process.spawnSync`](https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options).
+Returns the same result object as [`child_process.spawnSync`](https://nodejs.org/api/child_process.html#child_process_child_process_spawnsync_command_args_options). It's not possible to obtain a true interleaved `all` stream as `execa()` does, because `child_process.spawnSync` does not expose `stdout` and `stderr` to be combined until termination.
```

Comment:
I think the last sentence could be written better.

Reply:
Could this be better?
> It does not expose the additional `all` interleaved stream as `execa()` does because the underlaying sync implementation cannot guarantee a true interleaved output from `stdout` and `stderr`.

Reply:
> It does not have the `.all` property that `execa()` has because the underlying synchronous implementation only returns `stdout` and `stdout` at the end of the execution, so they cannot be interleaved.